satellite/payments/stripecoinpayments/service.go

```diff
@@ -583,20 +589,41 @@ func (service *Service) createInvoiceItems(ctx context.Context, cusID, projName
 
 	projectPrice := service.calculateProjectUsagePrice(record.Egress, record.Storage, record.Objects)
 
-	projectItem := &stripe.InvoiceItemParams{
-		Amount:      stripe.Int64(projectPrice.TotalInt64()),
-		Currency:    stripe.String(string(stripe.CurrencyUSD)),
-		Customer:    stripe.String(cusID),
-		Description: stripe.String(fmt.Sprintf("project %s", projName)),
+	projectItemBase := &stripe.InvoiceItemParams{
```

Comment:
I think you should change this variable to be value type, or later introduce of new item variables doesn't make sense as you change the same reference all the time